client/components/RoomList.jsx

```diff
@@ -0,0 +1,52 @@
+RoomList = React.createClass({
+
+  goToRoom(roomId, roomName) {
+    this.props.goToRoom(roomId, roomName);
+  },
+
+  goToNewRoom() {
+    this.props.setView("newRoom");
+  },
+
+  render() {
+    var that = this;
+
+    var loader =  (
+      <div className="ui active inverted dimmer">
+        <div className="ui text loader">
+          Loading messages
+        </div>
+      </div>
+    );
+
+    var joinedRooms = this.props.joinedRooms.map(function(room){
+      return <RoomListItem getUserCount={that.props.getUserCount} goToRoom={that.goToRoom} goToNewRoom={that.goToNewRoom} room={room} />;
+    });
+
+    var otherRooms = this.props.otherRooms.map(function(room){
+      return <RoomListItem getUserCount={that.props.getUserCount} goToRoom={that.goToRoom} goToNewRoom={that.goToNewRoom} room={room} />;
+    });
+
+    return (
+      <div className="roomList">
+        <div className="padded">
+          <div className="ui header">
+            Your channels
+          </div>
+          <div className="ui selection list celled">
+            { this.props.subsReady ? joinedRooms : loader}
+          </div>
+          <div className="ui header">
+            Other channels
+          </div>
+          <div className="ui selection list celled">
+            { this.props.subsReady ? otherRooms : loader}
+          </div>
+        </div>
+        <div className="ui fluid button primary newroom-btn" onClick={() => this.goToNewRoom()}>
```

Comment:
Takk for feedbaken! skal se over kommentarene og prøve å rette det tingene
før neste uke!
On 23 Mar 2016 12:26 p.m., "Jørgen Foss Eri" <notifications@github.com>
wrote:

> In client/components/RoomList.jsx
> <https://github.com/jorgeer/chatter-semantic/pull/1#discussion_r57136792>:
>
> > +      <div className="roomList">
> > +        <div className="padded">
> > +          <div className="ui header">
> > +            Your channels
> > +          </div>
> > +          <div className="ui selection list celled">
> > +            { this.props.subsReady ? joinedRooms : loader}
> > +          </div>
> > +          <div className="ui header">
> > +            Other channels
> > +          </div>
> > +          <div className="ui selection list celled">
> > +            { this.props.subsReady ? otherRooms : loader}
> > +          </div>
> > +        </div>
> > +        <div className="ui fluid button primary newroom-btn" onClick={() => this.goToNewRoom()}>
>
> Unødvendig å lage anonym funksjon om du ikke trenger noen argumenter.
> onClick={this.goToNewRoom}
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jorgeer/chatter-semantic/pull/1/files/c37b0650285459cf20fdab17c141f71b789cb694#r57136792>
>